pkgs/top-level/all-packages.nix

```diff
@@ -14497,6 +14497,8 @@ with pkgs;
 
   filecheck = with python3Packages; toPythonApplication filecheck;
 
+  files-to-prompt = with python3Packages; toPythonApplication files-to-prompt;
```

Comment:
This can also be done in pkgs/by-name. For that matter I think the entire package can go there, it doesn't really look like a library.